css/common/main.css

```diff
@@ -13,8 +13,10 @@ a {
 
 a:focus, a:hover {
   border-bottom: 1px solid;
-  background: #100170;
-  color: white;
+  background:white;
+  color:#2929a3;
+  transition-delay:0.2s;
```

Comment:
Don't we also need a transition timing for the colors @sheenachhabradev?

Reply:
Added transition property and timing-function.